doc/rst/language/spec/ranges.rst

```diff
@@ -892,8 +899,8 @@ resulting range is also ambiguously aligned.
 
 .. _Range_Slicing:
 
-Range Slicing
-~~~~~~~~~~~~~
+Range Slicing / Intersection
```

Comment:
I wouldn't make "Intersection" as prominent. I'm not too worried about this particular use, but if we do it here, I'd argue that we should do the same for Domain Slicing, which I'm worried about more. Because if we do that, we should do that for Array Slicing as well, which I am actively against. :)

Reply:
And I am fine with the statement "resulting range represents the *intersection* ... ". It can definitely help for someone new Ctrl+F'ing "intersection". But I suggest we don't equivocate "Slicing" and "Intersection".

Reply:
I wasn't making them equivalent in any way, but trying to come up with a section that might mean someone to someone looking for a particular capability.  I.e., if I'm scanning for how I might compute an intersection, this would catch my eye whereas "Range Slicing" wouldn't (and I don't think this needs to mean that _all_ slicing is necessarily intersection.  But I'm fine removing it as well.